contao/helper/exception.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+
+/**
+ * Class UnusedArgumentsException
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ */
+class UnusedArgumentsException extends Exception {}
+
+
+/**
+ * Class UnresolvableDependenciesException
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ */
+class UnresolvableDependenciesException extends Exception {}
```

Comment:
We're not using this anymore, as dependencies are handled by Composer. The class is just there for legacy compatibility if someone would have used it (for god knows why…).